cmd/cfgfiles/tapir-pop.yaml

```
@@ -0,0 +1,66 @@
+dnsengine:
+    addresses: [ ### EDIT Addresses (ip:port) to listen to for RPZ XFR requests ]
+    active: true
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 8 but found 4

(indentation)